Sample started code.py

```diff
@@ -1,54 +0,0 @@
-# -*- coding: utf-8 -*-
-import pandas as pd
-import numpy as np
-import MultiPyVu as mpv
-
-keithley = Keithley2400("GPIB::xxxx")
-
-max_current = 1,
-start_voltage = 1  ## Voltage to start with. 
-keithley.apply_voltage(voltage_range=start_voltage,compliance current = max_current)
-
-max_voltage = 1
-
-# increasing voltage linearly
-ramp_to_voltage(target_voltage=max_voltage, steps=30, pause=0.02)

```

Comment:
Similar to the comment above, where are you getting this function? I implement something like it in https://github.com/Quantum-Science-and-Technology-Lab/labber-wrapper but this is not quite the right name. Check out keithley_sweep: https://github.com/Quantum-Science-and-Technology-Lab/labber-wrapper/blob/main/experiments/keithley_sweep.py.